paket.dependencies

```diff
@@ -26,12 +26,15 @@ nuget Dapper 2.0.123
 nuget AsyncEnumerator 4.0.2
 nuget Destructurama.JsonNet 2.0.0
 
+nuget Microsoft.AspNetCore.Authentication.JwtBearer 6.0.11
```

Comment:
Isn't this a sub dependency of Basisregisters.AcmIdm and therefore indirectly included?